src/util/rewriteUrls.ts

```diff
@@ -46,8 +45,7 @@ export async function rewriteUrl(articleId: string, mw: MediaWiki, dump: Dump, l
     }
 
     if (!hrefProtocol && href.slice(0, 2) === '//') {
-        const wikiProtocol = urlParser.parse(mw.webUrl).protocol;
-        href = `${wikiProtocol}${href}`;
+        href = `${mw.protocol}${href}`;
         linkNode.setAttribute('href', href);
```

Comment:
Remove `href` variable and just assign `${mw.protocol}${href}`

Reply:
The reason of having the `href` variable is it is getting used in other places as well.